src/components/LoginPage/LoginForm.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { FC, useState, useEffect } from "react";
+import {
+    Box,
+    Button,
+    Grid,
+    InputLabel,
+    TextField,
+    Typography,
+} from "@mui/material";
+import { useNavigate } from "react-router-dom";
+
+import { spawnToast } from "../../utils/Toast";
+import { getUsers } from "../../api/users";
+import { userInfo } from "../../interfaces/interfaces";
+import { Login } from "../../interfaces/interfaces";
+
+const LoginForm: FC<Login> = ({
+    username,
+    password,
+    handleUsername,
+    handlePassword,
+}) => {
+    const navigate = useNavigate();
+    const [users, setUsers] = useState<userInfo[]>();
+    const getUsersAccounts = async () => {
+        try {
+            const result = await getUsers();
+            setUsers(result);
+        } catch (error) {
+            console.log(error);
+        }
+    };
+
+    useEffect(() => {
+        getUsersAccounts();
+    }, []);
+
+    useEffect(() => {
+        localStorage.setItem("authenticated", "false");
+        localStorage.setItem("user_type", "none");
+    }, []);
```

Comment:
Every time the component rerenders this block of code will run, are you sure you want this behaviour?